README.md

```diff
@@ -52,7 +52,9 @@ Add the following to your `~/.emacs` or `~/.emacs.d/init.el` file:
       where)))
 
 (let* ((refmt-bin (or (reason-cmd-where "refmt ----where")
-                      (shell-cmd "which refmt")))
+                      (shell-cmd "which refmt"))
+                      (or (reason-cmd-where "refmt ----where")
```

Comment:
I added another call to the "or function" (not sure how to describe it in English since I'm not super familiar with Elisp) instead of completely replacing the reference to "refmt" and I placed it after the call to the "or function" such that bsrefmt can be searched for as a fallback to cases where refmt doesn't exist in the user's fs.